### PR TITLE
Further support for more carts

### PR DESCRIPTION
**Pull request by lifehackerhansol**
TLDR:
- ~~Added `dsbize.c` (unfortunately needed for the M3 family)~~
- Added support for:
    - ~~M3 DS Real and derivatives~~
    - ~~M3i Zero (GMP-Z003)~~
    - EZ-Flash V
    - EZ-Flash Parallel (lol)
    - Games 'n' Music
    - `r4dspro.com`
        - For now this is a placeholder name, it refers to Acekard 2 derivatives that does not have a functional SDIO CMD25. It's done this way as I didn't want to gimp write speed for all Acekards just to unify this one guy.
        - There's only really two different models with this quirk I've seen in the wild, but the r4dspro is still being sold as of this writing.
    - EDGEi
        - Not to be confused with the iEDGE, it's just the usual AK2i derivative
    - Stargate 3DS (lol)

### CODE REVIEW

**Comment by asiekierka** [comment]
Can we split off the non-`dsbize` and `dsbize` carts into separate PRs? I'd still like to try and get rid of the "blobby" nature of replacing the whole header as `dsbize` does, if possible.

**Comment by lifehackerhansol** [comment]
> Can we split off the non-`dsbize` and `dsbize` carts into separate PRs? I'd still like to try and get rid of the "blobby" nature of replacing the whole header as `dsbize` does, if possible.

Removed M3 devices from this PR.

**Comment by asiekierka** [comment]
Thank you!
